src/app/features/Cart/cart/cart.component.html

```
@@ -2,7 +2,7 @@
 
     <h1 class="text-3xl font-bold mb-6">Shopping Cart</h1>
 
-    <app-toast *ngIf="showToast" [message]="message"></app-toast>
+    <app-toast *ngIf="showToast"[isSuccess]="isSuccess" [message]="message"></app-toast>
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)